### ISSUE TITLE
rust web development

### ISSUE THREAD

**Issue by wangjiawen2013** [fMeow/arangors#107]
Hi,
Is arangors still under maintanence ?

I want to to use ArangoDB in actix, an backend rust web framework:
https://github.com/actix/actix-web/discussions/3058#discussioncomment-6304643

**Comment by fMeow** [comment]
Yes, indeed. Though I might not be active in developing new features. Thanks to the AQL in arango, it's enough for our use case.
